### ISSUE TITLE
Removed and recreated files don't trigger add event when watched directory is symlinked

### ISSUE THREAD

**Issue by ezze** [paulmillr/chokidar#1064]
**Describe the bug**

The issue happens when watched directory is a symbolic link. `add` event for a file created within such kind of directory is triggered only once. If you remove and then recreate the file, there is no `add` event triggered anymore.

**Versions:**
 - Chokidar version: 3.5.0 (or any `^3.0.0`).
 - Node version: 14.15.1
 - OS version: Kubuntu 20.04.1

**To Reproduce:**

[Here](https://github.com/ezze/chokidar-issue-1064) is a repo to reproduce. Initially I created this repo considering that it's related to #917 but it's closed now. So I'm opening a new issue.

**Expected behavior**

Trigger `add` event for files being removed and added again as it was working in `chokidar@2`. You can just install it in my repo to see the expected behavior.

**Comment by paulmillr** [comment]
try `master`

**Comment by ezze** [comment]
Works with `master` for me. When can we expect it to be published to npm?

**Comment by paulmillr** [comment]
right away
